docs/Alerts/Add_Alerts/add_KPI_alert.md

```diff
@@ -37,3 +41,15 @@ You need to fill in the following fields:
 Consolidated [Reports need to be configured](/Alerts/Alert_Report_Settings.md) to receive them.
 :::
 
+## Examples of KPI alerts
+
+Example of email KPI alerts:
```

Comment:
```suggestion
### Email Alerts
```

Reply:
Example is already mentioned in the title. It's redundant to repeat it in the sub heading.